be/test/vec/core/column_complex_test.cpp

```diff
@@ -153,6 +181,21 @@ TEST_F(ColumnBitmapTest, ColumnBitmapReadWrite) {
     EXPECT_TRUE(bitmap.contains(1000000));
 }
 
+TEST_F(ColumnBitmapTest, OperatorValidate) {
+    auto column = _bitmap_type.create_column();
+
+    // empty column
+    check_serialize_and_deserialize(column);
+
+    // bitmap with lots of rows
+    const size_t row_size = 20000;
+    auto& data = assert_cast<ColumnBitmap&>(*column.get()).get_data();
+    data.resize(row_size);
+    check_serialize_and_deserialize(column);
+
+    check_field_type(column);
```

Comment:
Not only check the return type.
1. build a bitmap value and insert into the column 
2. get the filed value from column and check the value
3. check the type.